packages/ds-ext/src/messageHandlers/onUpdateDiagram.ts

```diff
@@ -5,6 +5,6 @@ export const onUpdateDiagram: MessageHandler = async ({
   event,
   document,
 }) => {
-  document.update(new TextEncoder().encode(event.diagram));
+  document.update(new TextEncoder().encode(JSON.stringify(event.diagram)));
```

Comment:
This is quite a change, might be related to section "Vscode broken"?

Reply:
Yes, I've standardized the data format returned by the various servers (jsServer, socketServer, and vscodeServer). Therefore, we need to recreate the .diagram.json file in ds-ext